cmd/istioctl/inject.go

```diff
@@ -107,10 +107,20 @@ Example usage:
 )
 
 func init() {
+	defaultHub := os.Getenv(inject.DefaultHubEnvVar)
```

Comment:
Long term having these variables is a mistake, because in the enterprise there will be some admins with the ability to create the manager and some mid-level ops who only have the authority to deploy their microservice.  Those folks won't know or care about the manager version.

Better would be for the manager on startup to write it's hub and tag to the istio configmap in its namespace.  By default if --hub and --tag aren't used is to have _istioctl kube-inject_ should find the manager version by asking the configmap.